### ISSUE TITLE
DeepLab Xception checkpoints are broken

### ISSUE THREAD

**Issue by jrbtaylor** [tensorflow/models#5742]
### System information
- **What is the top-level directory of the model you are using**: research/deeplab
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: Binary
- **TensorFlow version (use command below)**: 1.9
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**: 9.0
- **GPU model and memory**: Titan V
- **Exact command to reproduce**: see below

The following applies to both "xception65_cityscapes_trainfine" and "xception71_dpc_cityscapes_trainval" checkpoints from research/deeplab/model_zoo. If I build the graph with the included functions, i.e. 

`xception.xception_71(input_tensor, num_classes=30, is_training=False, global_pool=False, output_stride=8, scope='xception71')`

and try to restore from the checkpoints in the model zoo with `tf.train.Saver().restore(sess, ckpt_path)`, where ckpt_path is the unzipped checkpoint file, I get the following error:

> DataLossError (see above for traceback): Unable to open table file /segmentation.ckpt.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?
         [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]

This occurs for both the xception 65 and 71 models, and I was also unable to restore from the frozen_inference_graph.pb file (though I will exclude that because I would rather use the ckpt).

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
